src/renderer/src/i18n/locales/en-us.json

```
@@ -3119,7 +3119,14 @@
         "tip": "A summary of the reasoning performed by the model",
         "title": "Summary Mode"
       },
-      "title": "OpenAI Settings"
+      "title": "OpenAI Settings",
+      "verbosity": {
+        "high": "High",
+        "low": "low",
+        "medium": "middle",
+        "tip": "Control the level of detail in the model's output",
+        "title": "Level of detail"
```

```suggestion
        "tip": "Control the model's reasoning effort",
        "title": "Reasoning effort"
```
same as in [line 187](https://github.com/CherryHQ/cherry-studio/blob/5fdf2daba3e6355d9e187b6bb431a1082c74f13c/src/renderer/src/i18n/locales/en-us.json#L187)

(apply the same to the automatic translations)